### PR TITLE
support nullsafety

### PR DESCRIPTION
**Pull request by Ahmadre**
I use this package for my flutter_iconpicker package and my users needs nullsafety.

So I did:

`dart migrate --apply-changes`

to be able to support nullsafety.

There're no relevant changes, just the sdk version changed.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name: outline_material_icons
 description: Provides the new outline theme of material design icons from Google.
-version: 0.1.1
+version: 0.2.0-nullsafety
```
Comment (nilsreichardt) [inline pubspec.yaml]: Null safety is now stable so we can remove the prerelease

```suggestion
version: 0.2.0
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## [0.2.0-nullsafety] - 20/02/2021
```
Comment (nilsreichardt) [inline CHANGELOG.md]: Null safety is now stable so we can remove the prerelease

```suggestion
## [0.2.0] - 20/02/2021
```

**Review by nilsreichardt — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (Ahmadre) [reply pubspec.yaml]: And I was questening myself why others do this "-nullsafety" but now it's clear for me xD thx!

**Comment by emlynmac** [comment]
Any chance to get this merged soon? It's one of the dependencies holding me up from migrating to null-safety.

**Comment by nilsreichardt** [comment]
@emlynmac As a workaround you should be able to use this in your `pubspec.yaml`:

```yaml
outline_material_icons:
  git:
    url: https://github.com/Ahmadre/outline_material_icons
```

**Comment by Ahmadre** [comment]
> Any chance to get this merged soon? It's one of the dependencies holding me up from migrating to null-safety.

The outlined icons are now integrated in new flutter versions. Including rounded and so on.

But regarding the null safety problem: just add:

```
// @dart=2.9
```

on top of your main.dart file

**Pull request closed without merging**